unity-client/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/LoadWrapper/LoadWrapper_GLTF.cs

```diff
@@ -22,7 +23,11 @@ public override void Load(string targetUrl, Action<LoadWrapper> OnSuccess, Actio
 
             alreadyLoaded = false;
             Assert.IsFalse(string.IsNullOrEmpty(targetUrl), "url is null!!");
-            loadHelper = new RendereableAssetLoadHelper(this.entity.scene.contentProvider, entity.scene.sceneData.baseUrlBundles);
+
+            if (customContentProvider == null)
```

Comment:
This new field needs some additional test coverage. @AjimenezDCL can you onboard @Suduck on this?